app/views/step_by_step_pages/publish_or_delete.html.erb

```diff
@@ -25,7 +25,7 @@
 
 <div class="govuk-grid-row publish-or-delete">
   <div class="govuk-grid-column-full">
-    <% if @step_by_step_page.can_be_published? || @step_by_step_page.can_discard_changes? ||  @step_by_step_page.scheduled_for_publishing?%>
+    <% if @step_by_step_page.can_be_published? || @step_by_step_page.can_discard_changes? ||  @step_by_step_page.scheduled_for_publishing? || !@step_by_step_page.steps_have_content? %>
```

Comment:
This is getting quite long - any chance it can be encapsulated in a controller method? 🤔 

If not, there's a double-space before `@step_by_step_page.scheduled_for_publishing?` that can be fixed.

Reply:
This `if` determines whether or the grey background for the buttons is displayed and nothing else. Ideally presentation logic should be moved to a `Presenter` class an not the controller.

I think it's fine to leave this where it is for now as the tabbed layout is on it's way out, so it's not worth the effort of creating a new presenter.

I'll fix the double space problem and fix up.